tests/publiccloud/prepare_instance.pm

```diff
@@ -44,6 +44,7 @@ sub run {
     $instance->ssh_opts("");    # Clear $instance->ssh_opts which ombit the known hosts file and strict host checking by default
 
     $instance->network_speed_test();
+    $instance->check_cloudinit() if (cloudinit_enabled);
 
     # ssh-tunnel settings
     prepare_ssh_tunnel($instance) if (is_tunneled());
```

Comment:
and we will apply `permit_root_passwordless` second time in a row here ?

Reply:
ah sorry , please ignore this

Reply:
This is needed at least on EC2 as after `cloud-init clean` the "EC2 defaut configuration" gets reapplied and then openQA tries to `ssh` as `root` and instead of shell only message saying "Hey you, please use `ec2-user` instead" is printed.